test/third_party_tests/test_native_executor.py

```diff
@@ -38,7 +38,7 @@ def _create_table_in_native_database(self):
             """USE test_data_source {
                 CREATE TABLE test_table (
                     name VARCHAR(10),
-                    Age INT,
```

Comment:
I intentionally made this change so that we could test the corner case.

Reply:
Ah I missed that. Thanks for point it out.